apps/public-docsite/src/components/Site/Site.tsx

```diff
@@ -483,27 +474,42 @@ export class Site<TPlatforms extends string = string> extends React.Component<
       return;
     }
 
-    const platformKeys = platforms && (Object.keys(platforms) as TPlatforms[]);
```

Comment:
This is doing the same thing as before plus saving whatever it determines the current platform to be for use later (in the title)